src/utils/debounce.ts

```diff
@@ -0,0 +1,8 @@
+export const debounce = (f: (value: string) => void, delay: number) => {
+  let timerId: NodeJS.Timeout;
```

Comment:
do you really running the application server side so that the target in your case is Node JS?